diff -Nur POE-Component-Server-SimpleHTTP-1.30/Makefile.PL POE-Component-Server-SimpleHTTP-1.30.mod/Makefile.PL
--- POE-Component-Server-SimpleHTTP-1.30/Makefile.PL	2007-12-21 17:53:45.000000000 +0100
+++ POE-Component-Server-SimpleHTTP-1.30.mod/Makefile.PL	2008-01-07 22:45:33.183685924 +0100
@@ -21,17 +21,17 @@
 build_requires   'LWP::UserAgent'               =>    0;
 build_requires   'LWP::ConnCache'               =>    0;
 
-my $value = prompt( 'Do you want to test streaming ( requires POE::Component::Client::HTTP ) [y/N]?', 'N' );
+my $value = 'Y';
 build_requires   'POE::Component::Client::HTTP' =>    0 if $value =~ /^Y$/i;
 
 # Ask users if they want SSL support
-$value = prompt( 'Do you want SSL support ( requires POE::Component::SSLify ) [y/N]?', 'N' );
+$value = 'Y';
 
 # Add to the prereqs PoCo::SSLify?
 requires 'POE::Component::SSLify' => '0.04' if $value =~ /^Y$/i;
 
 # Ask users if they want PreFork support
-$value = prompt( 'Do you want PreFork support ( requires IPC::Shareable ) [y/N]?', 'N' );
+$value = 'Y';
 
 # Add to the prereqs IPC::Shareable?
 if ( $value =~ /^Y$/i ) {
